packages/amplify-provider-awscloudformation/src/push-resources.js

```diff
@@ -156,7 +156,14 @@ async function run(context, resourceDefinition) {
     // Store current cloud backend in S3 deployment bcuket
     await storeCurrentCloudBackend(context);
     await amplifyServiceManager.storeArtifactsForAmplifyService(context);
-
+    //check for auth resources and remove deployment secret for push
+    const authResources = resources.filter(resource => resource.category === 'auth');
```

Comment:
Make sure you filter of the provider - "awscloudformation" and the service "Cognito" as well here.